user/details/script.js

```diff
@@ -661,6 +664,262 @@ function showContent() {
+  userPr.appendChild(h2);
+  const table = createPrDetailsTable(prs);
+  const viewButton = createPrViewButton(prs);
+  userPr.append(table, viewButton);
+  return userPr;
+}
+
+function createPrViewButton(prs) {
+  const viewPRBtnDiv = createElement({
+    type: 'div',
+    classList: ['btn-wrapper'],
+  });
+  const viewPRBtn = createElement({
+    type: 'button',
+    classList: ['pr-view-btn'],
+  });
+  viewPRBtn.appendChild(createTextNode('View'));
+  viewPRBtn.addEventListener('click', (e) => {
+    window.open(prs.url, '_blank');
+  });
+  viewPRBtnDiv.appendChild(viewPRBtn);
+  return viewPRBtnDiv;
+}
+
+function createPrDetailsTable(prs) {
+  const container = createElement({
+    type: 'table',
+    classList: ['pr-details-table'],
+  });
+  const repoRow = createPrDetailsRow('Repository Name', prs.repository);
+  container.appendChild(repoRow);
+  const statusRow = createPrDetailsRow('Status', prs.state);
+  container.appendChild(statusRow);
+  const createdAtRow = createPrCreatedAt(
+    'Created At',
+    generateRemainingDays(prs.createdAt),
+    prs,
+  );
+  container.appendChild(createdAtRow);
+  const updatedAtRow = createPrUpdatedAt(
+    'Updated At',
+    generateRemainingDays(prs.updatedAt),
+    prs,
+  );
+  container.appendChild(updatedAtRow);
+  return container;
+}
+
+function createPrDetailsRow(head, data) {
+  const row = createElement({ type: 'tr', classList: ['pr-details-row'] });
+  const headElement = createElement({
+    type: 'td',
+    classList: ['pr-details-data'],
+  });
+  const headText = createElement({
+    type: 'h4',
+    classList: ['pr-details-head'],
+  });
+  headText.appendChild(createTextNode(head));
+  headElement.appendChild(headText);
+  const semiElement = createElement({ type: 'td', classList: ['colon'] });
+  semiElement.appendChild(createTextNode(':'));
+  const dataElement = createElement({
+    type: 'td',
+    classList: ['pr-details-data'],
+  });
+  dataElement.appendChild(createTextNode(data));
+  row.append(headElement, semiElement, dataElement);
+  return row;
+}
+
+function createHeadForPrDate(head) {
+  const headElement = createElement({
+    type: 'td',
+    classList: ['pr-details-data'],
+  });
+  const headText = createElement({
+    type: 'h4',
+    classList: ['pr-details-head'],
+  });
+  headText.appendChild(createTextNode(head));
+  headElement.appendChild(headText);
+  const semiElement = createElement({ type: 'td', classList: ['colon'] });
+  semiElement.appendChild(createTextNode(':'));
+  return { headElement, semiElement };
+}
+
+function createPrCreatedAt(head, data, prs) {
+  const row = createElement({ type: 'tr', classList: ['pr-details-row'] });
+  const { headElement, semiElement } = createHeadForPrDate(head);
+  const dataElement = createElement({
+    type: 'td',
+    classList: ['pr-details-data-created'],
+  });
+  dataElement.appendChild(createTextNode(data));
+  const tooltip = createElement({ type: 'span', classList: ['tooltiptext'] });
```

Comment:
see if we can use the `title` attribute of the HTML instead of this,
Here https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title